cmd/cert-checker/main.go

```diff
@@ -220,14 +220,14 @@ func (c *certChecker) checkCert(cert core.Certificate) (problems []string) {
 		// Run zlint checks
 		results := zlint.LintCertificate(parsedCert)
 		for name, res := range results.Results {
-			// ignore notices and warnings
-			if res.Status >= lints.Error {
-				prob := fmt.Sprintf("zlint %s: %s", res.Status, name)
-				if res.Details != "" {
-					prob = fmt.Sprintf("%s %s", prob, res.Details)
-				}
-				problems = append(problems, prob)
+			if _, ignore := ignoredLints[name]; ignore || res.Status <= lints.Pass {
```

Comment:
I think this breaks our deployability guarantee, with no config change this will change all existing notice/warning to failures, which require manual review by SRE. Might make sense to only enforce this change if the configured `IgnoredLints != nil`.

Reply:
I think it will be less confusing to just split this change:
* Leave the existing "allow info/warning"
* Add support for the ignore list

After this commit lands and the configs are updated, remove the "allow info/warning".

I'll update this branch accordingly.

Reply:
Resolved in 2cef3dc. I filed https://github.com/letsencrypt/boulder/issues/4273 for the follow-up.